ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -664,432 +642,369 @@ describe('WorkspaceEdit', () => {
   // regression test for RW-5132 - async version
   it('prevents multiple (async) Workspace creations via the same confirmation dialog', async () => {
     workspaceEditMode = WorkspaceEditMode.Duplicate;
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
+    renderComponent();
 
-    wrapper
-      .find('[data-test-id="review-request-btn-false"]')
-      .first()
-      .simulate('click');
-    await waitOneTickAndUpdate(wrapper);
+    userEvent.click(await screen.findByTestId('review-request-btn-false'));
 
     const numBefore = workspacesApi.workspaceOperations.length;
-    wrapper
-      .find('[data-test-id="workspace-save-btn"]')
-      .first()
-      .simulate('click');
-    await waitOneTickAndUpdate(wrapper);
-
-    wrapper
-      .find('[data-test-id="workspace-confirm-save-btn"]')
-      .first()
-      .simulate('click');
-    await waitOneTickAndUpdate(wrapper);
-
-    wrapper
-      .find('[data-test-id="workspace-confirm-save-btn"]')
-      .first()
-      .simulate('click');
-    await waitOneTickAndUpdate(wrapper);
-
-    wrapper
-      .find('[data-test-id="workspace-confirm-save-btn"]')
-      .first()
-      .simulate('click');
-    await waitOneTickAndUpdate(wrapper);
-
-    expect(workspacesApi.workspaceOperations.length).toEqual(numBefore + 1);
-    expect(mockNavigate).toHaveBeenCalledTimes(1);
-  });
-
-  it('supports waiting on access delays', async () => {
```

Comment:
reminder to follow up with removing "access delay" logic

Reply:
Good call: https://precisionmedicineinitiative.atlassian.net/browse/RW-12440